### ISSUE TITLE
Question about variable, losses_q

### ISSUE THREAD

**Issue by sailist** [dragen1860/MAML-Pytorch#52]
Hi authors, thanks for your repo, it's so elegant and help me much, but when I read the code, there is a question about it.

In [Maml forward function](https://github.com/dragen1860/MAML-Pytorch/blob/master/meta.py#L134), `losses_q`  is a list type variable, store some losses, but I found that only the last element be used finally.

```python
# end of all tasks
# sum over all losses on query set across all tasks
loss_q = losses_q[-1] / task_num
```

Then I read the shit-like [official implement] (https://github.com/cbfinn/maml/blob/master/maml.py#L135) code, and fount it sum over all losses
```python
self.total_loss1 = total_loss1 = tf.reduce_sum(lossesa) / tf.to_float(FLAGS.meta_batch_size)
```
I am very confused about it, and hope for a reply.

**Comment by sailist** [comment]
I understand, loss_q[i] means the i-th updated, but it contains all tasks loss, the optimizer will optimize the last updated loss on all tasks.Thanks for your code again :)
